### PR TITLE
[CARBONDATA-3120]Fixed the parent version error in MV module

### PR DESCRIPTION
**Pull request by Jonathan-Wei**
fix carbondata 1.5.1 Datamap's core and plan project, pom.xml version mismatch problem.
Update version 1.5.0 to 1.5.2-SNAPSHOT

Be sure to do all of the following checklist to help us incorporate 
your contribution quickly and easily:

 - [x] Any interfaces changed?
 No
 - [x] Any backward compatibility impacted?
 No
 - [x] Document update required?
NO
 - [x] Testing done
        Please provide details on 
        - Whether new unit test cases have been added or why no new tests are required?
        - How it is tested? Please attach test report.
        - Is it a performance related change? Please attach the performance test report.
        - Any additional information to help reviewers in testing this change.
 No
 - [x] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA. 
No

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

**Comment by Jonathan-Wei** [comment]
I download the apache-carbondata-1.5.1-rc1.tar.gz package. 
After decompression, the datamap mv/core mv/plan project was added to the main pom for compilation。

I check the pom file, parent.version is 1.5.0-snapshot. But apache-carbondata-1.5.1-rc1.tar.gz is 1.5.1.

**Comment by brijoobopanna** [comment]
retest this please

**Comment by CarbonDataQA** [comment]
Build Failed with Spark 2.2.1, Please check CI http://95.216.28.178:8080/job/ApacheCarbonPRBuilder1/1724/

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 2.3.1, Please check CI http://136.243.101.176:8080/job/carbondataprbuilder2.3/9772/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/1514/

File: datamap/mv/core/pom.xml [hunk 1]
```diff
@@ -22,7 +22,7 @@
   <parent>
     <groupId>org.apache.carbondata</groupId>
     <artifactId>carbondata-parent</artifactId>
-    <version>1.5.0-SNAPSHOT</version>
+    <version>1.5.1</version>
```
Comment (zzcclp) [inline datamap/mv/core/pom.xml]: please use 1.5.2-SNAPSHOT, not 1.5.1, the same as other module.

File: datamap/mv/plan/pom.xml [hunk 2]
```diff
@@ -22,7 +22,7 @@
   <parent>
     <groupId>org.apache.carbondata</groupId>
     <artifactId>carbondata-parent</artifactId>
-    <version>1.5.0-SNAPSHOT</version>
+    <version>1.5.1</version>
```
Comment (zzcclp) [inline datamap/mv/plan/pom.xml]: please use 1.5.2-SNAPSHOT, not 1.5.1, the same as other module.

File: datamap/mv/core/pom.xml (hunk 1 above)
Comment (Jonathan-Wei) [reply datamap/mv/core/pom.xml]: The package version I downloaded was 1.5.1.
apache-carbondata-1.5.1-rc1.tar.gz

Comment (zzcclp) [reply datamap/mv/core/pom.xml]: The package version you downloaded is from a tag, not from branch-1.5.
the version of all modules on branch-1.5 is 1.5.2-SNAPSHOT, when to be released, version manager will change version to 1.5.1 to tag a version package.

Comment (xubo245) [reply datamap/mv/core/pom.xml]: @ravipesala @sraghunandan Have you tested MV module before/when we release 1.5.1-rc1? Why didn't change it?

**Comment by xubo245** [comment]
please change the content：Update version 1.5.0 to 1.5.1
=》Update version 1.5.0 to 1.5.2-SNAPSHOT

**Comment by xubo245** [comment]
retest this please

**Comment by xubo245** [comment]
retest this please

**Comment by zzcclp** [comment]
retest this please

**Comment by sraghunandan** [comment]
retest this please

**Comment by ravipesala** [comment]
add to whitelist

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/1522/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.3.1, Please check CI http://136.243.101.176:8080/job/carbondataprbuilder2.3/9780/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://95.216.28.178:8080/job/ApacheCarbonPRBuilder1/1732/

**Comment by xubo245** [comment]
LGTM

**Comment by xuchuanyin** [comment]
This PR uses the version 1.5.2-snapshot, but the main pom uses 1.6.0-snapshot.
Is it intended?

**Comment by xubo245** [comment]
this PR is pushed to branch1.5.not master

**Comment by zzcclp** [comment]
LGTM

**Comment by Jonathan-Wei** [comment]
Fixed.

**Pull request closed without merging**